src/components/media_control/public/media-control.scss

```diff
@@ -166,14 +166,14 @@
         height: 100%;
         font-size: 20px;
 
-        &:before { content:"\\e001"; }
+        &:before { content:"\e001"; }
 
         &.playing {
-          &:before { content:"\\e002"; }
+          &:before { content:"\e002"; }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Colon after property should be followed by one space